components/progressbar/stories/progressbar.stories.js

```diff
@@ -51,18 +83,110 @@ export default {
 	args: {
 		rootClass: "spectrum-ProgressBar",
 		size: "m",
+		label: "Loading",
 		labelPosition: "top",
 		value: 0,
 		isIndeterminate: false,
+		showValueLabel: true,
 	},
 };
 
+/**
+ * By default, progress bars are determinate and have a blue fill that shows the progress. This component should have a label at the start of the track that gives context about the operation being performed. In rare cases where context is sufficient and an accessibility expert has reviewed the design, the label could be undefined. These progress bars should still include an “aria-label” or “aria-labelledby” in HTML, depending on the context. The label is always placed above the track.
```

Comment:
Some of this gets repeated in the "Without label" story, do you think it needs to be in both places?

Reply:
There's still a bit of repetition around L147:
> but in rare cases where context is sufficient and an accessibility expert has reviewed the design, the label could be undefined.